packages/components/InputPanel.tsx

```diff
@@ -0,0 +1,60 @@
+import { Flex, Grid, GridItem } from '@chakra-ui/react';
+
+import BaseButton from './BaseButton';
+
+const generateNumberList = (max = 10) => {
+  return Array(max)
+    .fill(null)
+    .map((_, i) => (i + 1) % max);
+};
+
+const numericButtons = generateNumberList().map((btnText) => {
```

Comment:
```suggestion
const numericButtons = generateNumberList().map((number) => {
```
nao gostei mto to nome btnText mas tmb n sei uma sugestao mto melhor kkkkk

Reply:
EU não sou muito fã de componentes dentro de componentes na real, eu faria oque o zenha indicou  e no HTML do componente principal já colocaria direto: 
```jsx
{numberList.map(() => {
   seus componentes aqui
)}

```

Reply:
Concordo com o Fra. Seria legal tira ro retorno desse cara, criar um componente  e ai sim usar ele no map, direto aonde ele é usado